src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__month calendar__month--start-day--sun calendar__month--length--29">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
Better to define which exact properties will be animated, because if you not define it, it will be applied to every property that can be animated